### ISSUE TITLE
Keyboard Shortcuts are now Secret

### ISSUE THREAD

**Issue by sqgl2** [Woundorf/foxreplace#239]
The key combinations (eg Ctrl-F9) no longer appear in the FireFox drop-down menu alongside the clickable options.

**Comment by Woundorf** [comment]
I guess you mean in the Tools menu. It's a limitation of WebExtensions, I can't make them appear there, at least not like Firefox shortcuts appear (at most I could put them in parentheses). However they appear in the toolbutton menu.

**Comment by sqgl2** [comment]
Yes I did mean the Tools menu and I am not sure if that menu is even standard anymore in FF (I may have had to tweak FF to make it appear). I have no idea what the "toolbutton menu" is. How will new users know? I am fine with it all but the more new users that adopt FR the more enthused you will be to keep developing (sorry I am being selfish :)
